tools/codegen/generate.bzl

```diff
@@ -89,30 +90,37 @@ def mixer_proto_library(
    # through the gogo_proto_* methods at the moment.
    proto_compile(**proto_compile_args)
 
+   mixer_gen_args += {
+       "name" : name + "_processor",
+       "src": name + "_proto.descriptor_set",
+       "importmap": dict(dict(MIXER_IMPORT_MAP, **GOGO_IMPORT_MAP), **importmap),
+       "out": name + "_processor.gen.go",
+       "out_template": name + "_tmpl.proto",
+   }
+
+   mixer_gen(**mixer_gen_args)
+
    gogoslick_args += {
       "name": name + "_gogo_proto",
-      "protos": protos,
+      "protos": [name + "_tmpl.proto"],
       "deps": MIXER_DEPS + GOGO_DEPS + deps,
       "imports": imports + MIXER_IMPORTS + PROTO_IMPORTS,
       "importmap": dict(dict(MIXER_IMPORT_MAP, **GOGO_IMPORT_MAP), **importmap),
       "inputs": inputs + MIXER_INPUTS + PROTO_INPUTS,
-      # "output_to_workspace": True,
+      # TODO HELP PLS. Need this to make build work. Is this bad ?
```

Comment:
I will have to play with this for a bit on my own to figure out the implications of all the generations bouncing around here. I have some ideas on how to improve, but it is going to take some experimentation.